### DISCUSSION TITLE
Might you adding .json editing ability to Notify?

### DISCUSSION THREAD

**Discussion by serrq** [aritra-tech/Notify#177] [Ideas]
I use Markor to do this, but I would uninstalling it to not overcrowding my device with "double" apps that do same things.

If you add .json editing I unistall Markor. 
Or maybe Notify is not the right choice for json files? 🤔

https://www.f-droid.org/packages/net.gsantner.markor/

**Comment by aritra-tech** [comment]
> I use Markor to do this, but I would uninstalling it to not overcrowding my device with "double" apps that do same things.
> 
> If you add .json editing I unistall Markor. Or maybe Notify is not the right choice for json files? 🤔
> 
> https://www.f-droid.org/packages/net.gsantner.markor/

Yeah, this feature is in my bucket list.
Will be working on that very soon.
